### ISSUE TITLE
Certificate Error when logging in to the Open Shift Registry

### ISSUE THREAD

**Issue by kdombros** [IBM/spm-kubernetes#58]
I am following the SPM Runbook for Code Ready Containers: https://ibm.github.io/spm-kubernetes/prereq/openshift/codeready-containers

When executing the following step: 

`docker login -u kubeadmin -p $(oc whoami -t) $(oc registry info --public)`

I get the following Certificate error: 

`Error response from daemon: Get https://default-route-openshift-image-registry.apps-crc.testing/v1/users/: x509: certificate signed by unknown authority`

Let me know if you have any thoughts or advice. Thanks!

**Comment by andreyzher** [comment]
Hi @kdombros 

You need to tell Docker to trust the certificate :

(assuming command is run on Linux)
```
CERT_PATH="/etc/docker/certs.d/$(oc registry info --public)"
mkdir -p $CERT_PATH
oc extract secret/router-ca -n openshift-ingress-operator --keys=tls.crt --to=$CERT_PATH
```

Docker loads all `*.crt` files under `/etc/docker/certs.d/<registryHost[:port]>` directories and uses them for verifying any self-signed certificates - you can read more about this and how this might look on other operating systems in the Docker [documentation](https://docs.docker.com/registry/insecure/#use-self-signed-certificates) (see item 3 in the section regarding use of self-signed certificates)

**Comment by kdombros** [comment]
Thanks @andreyzher - I will try/pursue this some time soon and let you know!

**Comment by ourboy** [comment]
Resolved in Runbook release for February 2021 #60
